hedera-node/src/main/java/com/hedera/services/records/TransactionRecordService.java

```diff
@@ -69,6 +69,9 @@ public void includeChangesToToken(Token token) {
 			}
 			txnCtx.setCreated(serialNumbers);
 		}
+		if(token.isNew()) {
```

Comment:
nit 1: formatting of the if statement - missing space
nit 2: I think that logically, it would be better to place the `isNew` check first - this way it would resemble the inline java docs and the lifecycle of the token
nit 3: Remove `Only the second and third is implemented at this time.` from the java docs